src/js/directives/directives.js

```diff
@@ -321,21 +321,22 @@ module.directive('rpNoPropagate', [function() {
 module.directive('rpSpinner', [function() {
   return {
     restrict: 'A',
-    link: function(scope, element, attr) {
+    require: '^form',
+    link: function(scope, element, attr, ctrl) {
       var spinner = null;
-      attr.$observe('rpSpinner', function(value) {
+      scope.$watch(ctrl.$name+'.'+element.attr('name')+'.$pending', function(pending) {
```

Comment:
Sorry, this was pep8 habit (no spacing between operators that are part of function arguments).